### PR TITLE
Upgrade opentelemetry to the latest version

### PR DESCRIPTION
**Pull request by RangelReale**
Fix source code API and paths that changed in the latest opentelemetry version

### CODE REVIEW

**Review by pirgeo — APPROVED** [review]
Thanks for updating this!

File: example/basic/main.go [hunk 1]
```diff
@@ -82,12 +82,26 @@ func main() {
 	_ = c.Start(context.Background())
 
 	meter := c.Meter("otel.dynatrace.com/basic")
-	vr := metric.Must(meter).NewFloat64Histogram("golang_histogram")
+	vr, err := meter.SyncFloat64().Histogram("golang_histogram")
+	if err != nil {
+		panic(err)
+	}
```
Comment (dyladan) [inline example/basic/main.go:88]: Was `Must` removed?

**Review by dyladan — APPROVED** [review]
LGTM thanks for taking the time

File: example/basic/main.go (hunk 1 above)
Comment (RangelReale) [reply example/basic/main.go:88]: It was removed on opentelemetry, there's no equivalent now. All examples available do this manual error checking now.

**Review by arminru — APPROVED** [review]
Thank you!

**Pull request merged**